### PR TITLE
Adds opacity property to particle systems

### PR DESCRIPTION
**Pull request by vlidholt**

### CODE REVIEW

File: packages/flutter_sprites/lib/src/particle_system.dart [hunk 1]
```diff
@@ -406,6 +409,14 @@ class ParticleSystem extends Node {
         } else {
           particleColor = colorSequence.colorAtPosition(particle.colorPos);
         }
+        if (opacity != 1.0 || true) {
```
Comment (abarth) [inline packages/flutter_sprites/lib/src/particle_system.dart]: `|| true` ?

File: packages/flutter_sprites/lib/src/particle_system.dart [hunk 2]
```diff
@@ -406,6 +409,14 @@ class ParticleSystem extends Node {
...
+          particleColor = new Color.fromARGB(
```
Comment (abarth) [inline packages/flutter_sprites/lib/src/particle_system.dart]: particleColor.withAlpha(...)

**Comment by abarth** [comment]
LGTM

File: packages/flutter_sprites/lib/src/particle_system.dart (hunk 1 above)
Comment (vlidholt) [inline packages/flutter_sprites/lib/src/particle_system.dart]: Oops, good catch, debug code still in there

File: packages/flutter_sprites/lib/src/particle_system.dart (hunk 2 above)
Comment (vlidholt) [inline packages/flutter_sprites/lib/src/particle_system.dart]: Thanks, will change these

**Pull request merged**